ckan/lib/base.py

```diff
@@ -232,6 +230,44 @@ def set_pylons_response_headers(allow_cache):
     return render_template
 
 
+def _allow_caching(extra_vars, cache_force=None):
```

Comment:
`extra_vars` is not needed as a parameter since we are not using it in the code.